utils/generate_confusables.py

```diff
@@ -79,19 +82,30 @@ def main(args=sys.argv):
 //
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
+//
+////////////////////////////////////////////////////////////////////////////////
+// WARNING: This file is manually generated from
+// utils/UnicodeData/confusables.txt and should not be directly modified.
+// Run utils/generate_confusables.py to regenerate this file.
+////////////////////////////////////////////////////////////////////////////////
 '''
         f.write(header)
         f.write("//===----------------------------------------------------")
         f.write("------------------===//\n\n")
-        f.write("// CONFUSABLE(CONFUSABLE_POINT, BASEPOINT)\n\n")
-        for (confused, expected) in pairs:
-            f.write("CONFUSABLE(" + confused + ", " + expected + ")\n")
+        f.write("// CONFUSABLE(CONFUSABLE_POINT, CONFUSABLE_NAME, BASE_POINT, BASE_NAME)\n\n")
+        for (confused, confusedName, expected, expectedName) in pairs:
+            # Ad-hoc substitutions for clarity
+            mappings = { "Solidus": "Back Slash", "Reverse Solidus": "Forward Slash" }
+            newExpectedName = expectedName
+            if expectedName in mappings:
+                newExpectedName = mappings[expectedName]
+            f.write("CONFUSABLE(" + confused + ", " + '"' + confusedName + '"' + ", " + expected + ", " + '"' + newExpectedName + '"' + ")\n")
```

Comment:
This probably fails `validation-test/Python/python_lint.swift`. Try `@swift-ci Python lint`

Reply:
I have fixed it locally, but will run CI to double-check!